xml/releasenotes50.xml

```diff
@@ -19,7 +19,9 @@
  <xi:include href="newfeatures50.xml"/>
  <xi:include href="fixedissues50.xml"/>
  <xi:include href="knownissues50.xml"/>
- <xi:include href="unsupported_features.xml"/>
+ <!-- csymons 2018-05-25:
+     <xi:include href="unsupported_features.xml"/>
+-->
```

Comment:
It is unlikely that we will ever ship the HOS 5 release notes again -- what is the point of this change?

Reply:
Reverted.
I guess that this deletion and all places where it is used should just be removed. No need for releasenotes50.xml RN501 or RN502. They are included in introduction.xml, which is included in book.hos-index.xml.  Can I remove all of them?